### PR TITLE
Add client libraries for MediaWiki

### PR DESCRIPTION
**Pull request by FranklinYu**
Thanks for contributing to the Ruby Toolbox catalog! <3

Before submitting your pull request:

* If you are submitting a github-based project, please verify the project is not packaged as a rubygem
* Please make sure the projects are listed in alphabetical order
* Make sure the CI build passes, we validate your proposed changes in the test suite :)

### note

* I’m not sure about the order of hyphen and underscore; ASCII order is assumed.
* Data from <https://www.mediawiki.org/wiki/API:Client_code#Ruby>.

Blocked by #109.

### CODE REVIEW

**Comment by colszowka** [comment]
Thank you! Considering https://github.com/rubytoolbox/rubytoolbox/issues/131 I think it would make sense to fork this en bloc into it's own category. Would you mind moving this into a new `API Clients (group) -> MediaWiki Clients` category please? Thanks!

**Comment by colszowka** [comment]
@FranklinYu Now that #109 is finally out of the way, would you mind updating this?

I would propose to use the category name `MediaWiki Clients`, permalink `MediaWiki_Clients.yml` and "Libraries for interacting with MediaWiki, the open source Wiki platform that is also used on Wikipedia" as the category description

**Comment by FranklinYu** [comment]
I'm not accessible to my computer now. I would do it tonight (EST).

**Comment by colszowka** [comment]
Perfect :) No rush, thanks!

**Comment by FranklinYu** [comment]
Ready for review.

File: catalog/HTTP_API_Clients/MediaWiki_Clients.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: MediaWiki Clients
+description: Libraries for interacting with MediaWiki, the open source Wiki platform that is also used on Wikipedia
+projects:
+  - infoboxer
+  - mediawiki-butt
+  - mediawiki-gateway
+  - mediawiki_api
+  - mediawiktory
+  - mixpanel_client
```
Comment (colszowka) [inline catalog/HTTP_API_Clients/MediaWiki_Clients.yml]: This one seems incorrect, it's not related to mediawiki :)

**Comment by colszowka** [comment]
@FranklinYu Cool, thank you! There is one wrong gem that probably got in accidentally, can you please remove it? Then good to go

**Comment by FranklinYu** [comment]
Amended the commit.

**Comment by colszowka** [comment]
Great, thank you @FranklinYu !

**Pull request merged**